app.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Botpress",
+  "description": "The Conversational Platform with built-in language understanding (NLU), beautiful graphical interface and Dialog Manager (DM).",
+  "repository": "https://github.com/botpress/botpress",
+  "formation": {
+    "web": {
+      "quantity": 1
+    }
+  },
+  "env": {
+    "BPFS_STORAGE": {
+      "description": "Tells botpress to use an external database (postgres). Should be left at default.",
+      "value": "database"
+    }
```

Comment:
This produces a textbox in the one click menu which is less than ideal. The value shouldn't be configurable. The [documentation ](https://devcenter.heroku.com/articles/app-json-schema#env) doesn't provide a way to hide these so I added a description as a work-around.